client/web/web.go

```diff
@@ -202,6 +206,23 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 }
 
 func (s *Server) serve(w http.ResponseWriter, r *http.Request) {
+	// serve HTTP 204 on /ok requests as connectivity check
+	if r.Method == httpm.GET && r.URL.Path == "/ok" {
+		w.WriteHeader(http.StatusNoContent)
+		return
+	}
```

Comment:
I wonder if we should put this after `requireTailscaleIP` and specifically also only serve it in "manage" mode? That way we only respond to the request if made to the Tailscale IP. I guess they really couldn't hit this at all if they didn't make the request over Tailscale anyways, but feels a little safer...

Reply:
yeah, that should be fine.  I'll do that.